src/stdatamodels/jwst/transforms/models.py

```diff
@@ -1435,48 +1521,59 @@ def compute_refraction_index(lam, temp, tref, pref, pressure, kcoef, lcoef, tcoe
 
         # Convert to microns
         lam = np.asarray(lam * 1e6)
-        KtoC = 273.15  # kelvin to celsius conversion
+        KtoC = 273.15  # kelvin to celsius conversion  # noqa: N806
```

Comment:
can this be replaced by `k_to_c` or similar here?

Reply:
It could. However this line shows no test coverage. I'd rather err on the side of caution of add the noqa. New code can follow the convention but I'd like to see test coverage first (which is beyond the scope of this PR).

Reply:
Sounds good, see my other comment about what issue(s) to open to ensure this gets tracked

Reply:
How does https://github.com/spacetelescope/stdatamodels/issues/388 look?

Reply:
Thanks, sounds great to me